includes/class-wc-admin-reports-interval.php

```diff
@@ -474,4 +474,38 @@ public static function intervals_missing( $expected_interval_count, $db_records,
 		}
 	}
 
+	/**
+	 * Normalize "*_between" parameters to "*_min" and "*_max".
+	 *
+	 * @param array        $request Query params from REST API request.
+	 * @param string|array $param_names One or more param names to handle. Should not include "_between" suffix.
+	 * @return array Normalized query values.
+	 */
+	public static function normalize_between_params( $request, $param_names ) {
+		if ( ! is_array( $param_names ) ) {
+			$param_names = array( $param_names );
+		}
+
+		$normalized = array();
+
+		foreach ( $param_names as $param_name ) {
+			if ( empty( $request[ $param_name . '_between' ] ) ) {
+				continue;
+			}
+
+			$min_param = $param_name . '_min';
+			$max_param = $param_name . '_max';
+			$range     = explode( ',', $request[ $param_name . '_between' ] );
+
+			if ( isset( $range[0] ) && is_numeric( $range[0] ) ) {
```

Comment:
Same question here.  I would expect a count of 2 to be given or assume this would be malformed and expected the client side to validate having both fields.  But maybe leaving off the front-end validation (as it is now) is easier for users.

Reply:
In https://github.com/woocommerce/wc-admin/pull/1260/commits/ad9cbf4f6e3264e2d06fbea2f80778fc35eb4e35, I added validation and more normalization logic - the lower numeric value is always set to `_min`.